console/frontend/src/main/frontend/src/app/components/pages/side-navigation.directive.spec.ts

```diff
@@ -1,8 +1,49 @@
+import { Component } from '@angular/core';
 import { SideNavigationDirective } from './side-navigation.directive';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+@Component({
+  // standalone: true,
+  template: `
+    <ul>
+      <li>
+        <a>Category 1</a>
+        <ul>
+          <li><a>Item 1</a></li>
+          <li><a>Item 2</a></li>
+          <li><a>Item 3</a></li>
+        </ul>
+      </li>
+      <li>
+        <a>Category 2</a>
+        <ul>
+          <li><a>Item 1</a></li>
+          <li><a>Item 2</a></li>
+          <li><a>Item 3</a></li>
+        </ul>
+      </li>
+    </ul>
+  `,
+  imports: [SideNavigationDirective],
+})
+class TestComponent {}
 
 describe('SideNavigationDirective', () => {
-  it('should create an instance', () => {
-    const directive = new SideNavigationDirective();
-    expect(directive).toBeTruthy();
+  let fixture: ComponentFixture<TestComponent>;
+  beforeEach(() => {
+    fixture = TestBed.configureTestingModule({
+      imports: [],
+      declarations: [TestComponent, SideNavigationDirective],
+    }).createComponent(TestComponent);
+
+    fixture.detectChanges(); // initial binding
+  });
+
+  // TODO figure out a way to test metismenu
+  // https://angular.io/guide/testing-attribute-directives
+
+  // temporatory, remove when making actual tests
```

Comment:
When will this be done?

Reply:
I changed my mind in trying to unit test this.
The [side-navigation directive](https://github.com/frankframework/frankframework/blob/issue/6336-console-write-unit-tests-for-pipes-directives-utility-functions/console/frontend/src/main/frontend/src/app/components/pages/side-navigation.directive.ts) is only a call to the start the metismenu jQuery plugin.
Its not worth it to unit test this because its not completely our responsibility that their code works but also metismenu will have to be replaced in the future because jQuery is going to be removed hopefully sooner rather than later